### PR TITLE
Add money slides

### PR DESCRIPTION
**Pull request by golmit**
# Description

* Add the initial version of the slide deck about monetary theory basics including all assets.
* This change requires the colortbl which is licensed under LPPL.
* This change requires the pgfplots package which is licensed under gpl3+.
* The packages colortbl and pgfplots are added to config.tex

## Type of Change

Please delete options that are not relevant.

- [x] Major feature (New files or assets, large changes to files)

# Checklist:

- [x] I have performed a self-review of my own changes
- [x] I have made sure that any changed LaTeX files compile properly
- [x] I have assigned at least one reviewer

### CODE REVIEW

**Review by Chainomics — APPROVED** [review]
Great job @golmit! I might be biased in terms of the structure because it is relatively close to last year's lecture - so let's wait for the other reviewers before I rebase and merge. In any case, the slides are exceptionally well done and much clearer than last year!

**Comment by dariothuerkauf** [comment]
Looks very good! I think last years slides already had a very good structure and gave a great overview. Otherwise I have nothing to add. 💯

**Review by dariothuerkauf — APPROVED** [review]

**Comment by golmit** [comment]
I added three minor fixes:
- 2f03971: use caption instead of footnotesize
- d6fda99: include verifiability on money properties slide
- ffca1b7: use uncover instead of only for better-looking PDFs
Just a heads up before you merge @Chainomics

**Pull request merged**
